src/components/organisms/Breadcrumbs/index.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { Breadcrumbs } from './index';
+
+export default {
+  component: Breadcrumbs,
+  title: 'organisms/Breadcrumbs',
+} as Meta;
+
+const Template: Story = () => <Breadcrumbs items={[{ link_url: "a", text: "a" }, { link_url: "aaaa", text: "aaaa" }, { link_url: "aaaa", text: "aaaa" }, { text: "none" }]} />;
```

Comment:
コードをパッと見で少し見づらくなったので、分けてあげると良さそうかなと思いました。

```suggestion

const props = [
  { link_url: "a", text: "a" },
  { link_url: "aaaa", text: "aaaa" },
  { link_url: "aaaa", text: "aaaa" },
  { text: "none" }
];

const Template: Story = () => <Breadcrumbs items={props} />;
```

Reply:
link_urlが重複していたのでそれを修正して反映させました
https://github.com/yumemiHidetoKumakura/yumeshop-frontend/pull/9/files